tests/mockProfile.spec.js

```diff
@@ -0,0 +1,47 @@
+import { expect, test } from "@playwright/test";
+
+test("Mock profile", async ({ page }) => {
+  const sigInBtn = page.locator(".header_signin");
+  const emailInput = page.locator("#signinEmail");
+  const passwordInput = page.locator("#signinPassword");
+  const loginBtn = await page.locator(
+    'button.btn.btn-primary:has-text("Login")'
+  );
+  const profileBtn = await page.locator(
+    'a.btn-sidebar.sidebar_btn:has-text("Profile")'
+  );
+
+  await page.goto("/");
+
+  await sigInBtn.click();
+  await emailInput.fill("a.filatov1@yopmail.com");
+  await passwordInput.fill("Aaa!1234");
+
+  await loginBtn.click();
+
+  await page.waitForTimeout(10000);
+
+  await page.route("/api/users/profile", async (route) => {
+    const modifiedResponse = {
+      status: "ok",
+      data: {
+        userId: 131365,
+        photoFilename: "modified-user.png",
+        name: "Andrew",
+        lastName: "MockedUser",
+      },
+    };
+
+    await route.fulfill({
+      contentType: "application/json",
+      body: JSON.stringify(modifiedResponse),
+    });
+  });
+
+  await profileBtn.click();
+
+  const profileName = await page.locator(".profile_name.display-4");
+  const textContent = await profileName.textContent();
+
+  expect(textContent).toContain("Andrew MockedUser");
```

Comment:
use https://playwright.dev/docs/api/class-locatorassertions#locator-assertions-to-have-text